src/components/Character/styles.scss

```diff
@@ -1,19 +1,147 @@
 .wrapper {
-	font-size: 1em;
+	position: absolute;
+	top: 0;
+	width: 100%;
+	height: 100%;
 
 	&[aria-hidden="true"]{
 		opacity: 0;
+		pointer-events: none
 	}
 }
-	.button {
-		&.next {
-			font-size: 1em;
+
+	.container {
+		display: flex;
+		width: 100%;
+		height: 100%;
+		flex-direction: column;
+		justify-content: flex-end;
+		align-items: center;
+		font-size: 1.25rem;
+		--padding_container: 10vw;
+	}
+
+		.content {
+			position: relative;
+			width: 75%;
+			height: 50%;
+			z-index: 1;
 		}
-		&.previous {
-			font-size: 1em;
+	
+			.text {
+				overflow-y: scroll;
+			}
+
+			.name {
+				margin-bottom: 0.4em;
```

Comment:
all these margins under the elemnts here seem to vary widly - you sure the margins are related to the size of the elements themselves, or are they all actually based on `rem`? Do you get cleaner numbers if you go through the body size?

Reply:
I didn't really get any relationships that I could use :(